.github/workflows/python-release-patch.yml

```diff
@@ -78,3 +78,14 @@ jobs:
           PYPI_USERNAME: ${{ secrets.PYPI_USER }}
           PYPI_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
         run: poetry publish -u $PYPI_USERNAME -p $PYPI_PASSWORD -v -n
+
+      - name: Bump repo version
+        run: |
+          git clean -fxd # Clear all the changes
+          poetry version ${{ env.short_version }}.dev0
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v4
+        with:
```

Comment:
I think we can remove the 

```yaml
on:
  push:
    branches: [release]
```

trigger and make it only manually.

Reply:
> This means deleting the release branch and making the releases happen from main branch (I am OK with this)

@chamini2 I thought the release branch was useful (for only publishing certain fixes / revisions), especially in these times when we are landing a lot of new features that might not be completed. I'd be OK in either way, but if the `release` is useful we can still use a bot account to make the PRs. WDYT?

Reply:
I have been thinking and I do not think we should be doing a PR after we deployed. My thinking is that if we trigger a release, it will commit the changes right away to the branch, since we do not want our code to be "out-of-sync" with the released code.

Then... If we keep the `release` branch around and switch to pushing instead of pull-request. The push will happen to `release` and the `main` branch will be out-of-sync from `release` (`release` would be ahead). I think in that case, making a PR to `main` from `release` would make sense.

Reply:
I think in either case (on push to `release` branch & workflow run), it makes sense to do the bump in the `main`. Because it does not belong to the `release` branch (that branch only includes released stuff) as is, when we are releasing we can just take that commit as well as the others we want to cherry-pick.